### PR TITLE
created google news, image and video support

### PR DESCRIPTION
**Pull request by zuffik**
References: #110

### CODE REVIEW

**Comment by cipriancaba** [comment]
Is this something that could go live @iamvishnusankar ?

**Review by georgebutter — COMMENTED** [review]
@zuffik 
Curious about what this addition is for?

File: .gitignore [hunk 1]
```diff
@@ -68,3 +68,4 @@ junit.xml
 tsconfig.tsbuildinfo
 **/public
 **/public
+.idea
```
Comment (georgebutter) [inline .gitignore:71]: @zuffik I am curious about what this change is for?

**Review by georgebutter — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (zuffik) [reply .gitignore:71]: To prevent config files from jetbrains IDEs (mine is WebStorm) to being tracked in git.

**Comment by JamesSingleton** [comment]
Yeah this would be a nice addition to be able to create news sitemaps

**Comment by liAmirali** [comment]
Hey, any updates on this PR progress? Is there any problem with merging it?

**Comment by rokoroku** [comment]
@iamvishnusankar any updates?

**Comment by StewartJ-SSG** [comment]
We're using GoogleNews sitemaps and using next-sitemap, it would be real handy if this PR could get merged. What's the holdup @iamvishnusankar ?

**Comment by z0lo13** [comment]
@iamvishnusankar @georgebutter Is anything missing for a merge?

**Comment by georgebutter** [comment]
@z0lo13 , no, but only @iamvishnusankar  can merge it.

**Comment by iamvishnusankar** [comment]
@zuffik Thanks for this PR. Can you please fix the test errors? 

Error logs: https://github.com/iamvishnusankar/next-sitemap/actions/runs/4352877388/jobs/7606207567#step:6:41

**Comment by iamvishnusankar** [comment]
@zuffik Run `yarn format` on project root to resolve the formatting errors

**Comment by zuffik** [comment]
@iamvishnusankar done

**Review by iamvishnusankar — APPROVED** [review]
@zuffik Thanks for the PR! Approved 🙏

**Pull request merged**